Project.toml

```
@@ -32,7 +32,7 @@ AbstractAlgebra = "0.45.1"
 AlgebraicSolving = "0.9.0"
 Compat = "4.13.0"
 Distributed = "1.6"
-GAP = "0.13.4"
+GAP = "0.13.4, 0.14"
```

```suggestion
GAP = "0.14"
```
I would prefer to not allow both minor versions, as CI won't notify us once we break compatibility with 0.13.x.
And I don't see any good reason to have both in there (except for not needing to wait for a Hecke release)

---

```suggestion
GAP = "0.14.1"
```